src/main/java/io/github/mooy1/simpleutils/implementation/blocks/Elevator.java

```diff
@@ -57,8 +57,14 @@ public boolean isSynchronized() {
             @Override
             public void tick(Block block, SlimefunItem slimefunItem, Config config) {
                 if (SimpleUtils.slimefunTickCount() % 16 == 0 && block.getY() > block.getWorld().getMinHeight()) {
+                    final List<Material> blackList = List.of(
```

Comment:
Can the list please be declared as a public constant in the class, not within the method, and I see no reasonable need to include Netherite and Diamond blocks? They are valid building materials and as such should be usable for this method, no?

Reply:
Netherite block can be used to build a cheap Foundry with only 2 Netherite block and 4 elevators instead of 6 as can be observed at #9.

Reply:
As Riley mentioned they are easy to get and valid blocks for this purpose, the fix for that specific fringe case would be for Fluffy to ensure the blocks are not SlimefunItems (or slimefun perhaps if it is a traditional multiblock, not checked the code here).

Reply:
Ok I agree.
Removed others and left only spawners in the list.